src/providers/openai_realtime.py

```
@@ -779,6 +834,14 @@ async def stop_session(self):
             self._keepalive_task = None
             self._greeting_vad_task = None
             self._background_tasks.clear()
+            # Unblock any pending function_call handlers and drop their sentinels so they
+            # exit cleanly instead of waiting for a response.done that will never arrive.
+            try:
+                for _evt in self._response_done_events.values():
+                    _evt.set()
+                self._response_done_events.clear()
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)